### ISSUE TITLE
[plot3d] Mouse click not detected under the top-right axes

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#2361]
While moving an object on the screen with the rotation tool, i notice that the object under the circle containing the axes can't be grabbed when we click on it.

I don't know if it is a bug or what.

**Comment by vallsv** [comment]
The zone is not only the circle, but the bbox of the circle

**Comment by t20100** [comment]
Yes it's a bug, I guess the overlay circle (and the surrounding bounding box is updating the depth buffer at its position.
